app/uk/gov/hmrc/mtdsatestsupportapi/models/request/deleteTestBusiness/DeleteTestBusinessRequest.scala

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
This file was copied from a dependent PR ( see #46 ). So it might be a good idea to merge that PR first before merging this file.